dbt_subprojects/solana/models/_sector/dex/bot_trades/solana/_schema.yml

```
@@ -777,3 +777,25 @@ models:
       - check_bot_trades_seed:
           seed_file: ref('sanji_solana_trades_seed')
           blockchain: solana
+
+  - name: fasol_solana_bot_trades
+    meta:
+      blockchain: solana
+      sector: dex
+      project: fasol
+      contributors: whale_hunter
```

Maybe you can add your credit:
```suggestion
      contributors: whale_hunter, clizzard7
```